server.js

```diff
@@ -9,17 +9,35 @@ const server = express();
 // allow server to parse JSON bodies from POST/PUT/DELETE requests
 server.use(bodyParser.json());
 
+// TODO: DONE - write your server code here
 
+// get Bear model
+const Bear = require('./models');
 
+// POST /bears -- accepts bear
+server.post('/bears', (req, res) => {
+  const newBear = new Bear(req.body);
```

Comment:
You should never just pass in `req.body` imagine if someone sent up some corrupted data? You'd just be posting that to your Mongo instance. Obvs, there are validation measures you can put in place, but you should pull of the data you want from `req.body` and just save that data instead as best practices!